src/dymaptic.GeoBlazor.Core/Scripts/geometry.ts

```diff
@@ -0,0 +1,47 @@
+﻿import { arcGisObjectRefs } from "./arcGisJsInterop";
+let dotNetRef: any = null;
+
+export function initialize(dotNetReference: any, viewId: string, geometryId: string, geometryType: string): void {
```

Comment:
A few thoughts here.
- Unless I'm missing something, your main logic is commented out?
- We already have a listener to click events on the view, and more explicit ones coming in #102. I think @morehavoc's suggestion was to expose `hitTest` in the C# as a separate message on `MapView`. That way, you would chain in C#: in a click event callback, pass the event to `view.HitTest(point)` and get a response. Now, I don't mind us wrapping up that logic _as well_ as exposing it, but we should expose it first, then write a convenience method around it, something like `public List<Geometry> OnClickGeometries` in `MapView`.
- In general, I am aware that `MapView.razor.cs` and `arcGisObjectRefs.ts` are getting cumbersome and large, so I'm not opposed to adding new `.ts` files and spreading out the logic. But as you discovered when fighting with imports, it does have a cost associated.
- If there's a good use case for each geometry to track their own selection, rather than returning a list from the view, let's keep going with this. However, like I said above, I think we need to implement a view-level response first, and then maybe have that response trigger the geometry-level response?